sessions/react-data-fetching/star-wars/pages/index.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+import Layout from "../components/Layout";
+
+export default function HomePage() {
+  return (
+    <Layout>
+      <h1>React Data Fetching: Star Wars</h1>
+      <Link href="characters/luke-skywalker">Link to first character</Link>
```

Comment:
How do you feel about adding some links to characters to the homepage? 

This way students easier understand what to do and can quickly test the dynamic routes.

We should mention in the README that only the first link works in the beginning.

```
      <ul>
        <li>
            <Link href="/characters/1">Luke Skywalker</Link>
        </li>
        <li>
            <Link href="/characters/2">C-3PO</Link>
        </li>
        <li>
            <Link href="/characters/3">R2-D2</Link>
        </li>
        <li>
            <Link href="/characters/4">Darth Vader</Link>
        </li>
      </ul>
```